ui-admin/package.json

```diff
@@ -36,7 +36,6 @@
     "@types/react": "^18.0.18",
     "@types/react-dom": "^18.0.6",
     "@types/react-plotly.js": "^2.6.0",
-    "@types/react-table": "^7.7.12",
```

Comment:
This package defines types for an older version of `@tanstack/react-table`. v8 packages its own types.